src/main/java/core/basesyntax/NumberBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberBall {
+    public int getRandomNumber() {
```

Comment:
The method 'getRandomNumber' should also have an access modifier to define its visibility explicitly.